stix_shifter_modules/guardium/stix_transmission/guard_utils.py

```diff
@@ -137,26 +138,19 @@ def handle_report(self, report_name, params, index_from, fetch_size):
         return response
 
 
-    def handle_qs(self, category, params, filters, index_from, fetch_size):
+    def handle_qs(self, params, index_from, fetch_size):
         # -------------------------------------------------------------------------------
         # QS
         # -------------------------------------------------------------------------------
         if not self.fields:
              self.get_field_titles()
         
         results = ""
-        params_set = {"category":"{0}".format(category), "startTime": "{0}".format(params["startTime"]), "endTime": "{0}".format(params["endTime"]), \
-             "fetchSize": "{0}".format(int(fetch_size-1)), "firstPosition": "{0}".format(int(index_from-1)), "inputTZ":"UTC"}
-        if filters:
-            params_set["filters"] = "{0}".format(filters)
-        else:
-            if params:
-                if "filters" in params:
-                   params_set["filters"] = "{0}".format(params["filters"])
-                else: 
-                    params_set["filters"] = ""
-        all_params = {**params_set, **params}
-        json_dump = json.dumps(all_params)
+        params["fetchSize"] = int(fetch_size-1)
+        params["firstPosition"]=int(index_from-1)
+        params["inputTZ"]="UTC"
+
+        json_dump = json.dumps(params)
         rest_data = str(json.loads(json_dump))
```

Comment:
similar to https://github.com/opencybersecurityalliance/stix-shifter/pull/577/files#r608212762

Reply:
Removed redundant json conversion